entity-service/src/test/kotlin/com/egm/stellio/entity/service/SubscriptionHandlerServiceTests.kt

```diff
@@ -84,6 +84,52 @@ class SubscriptionHandlerServiceTests {
         confirmVerified()
     }
 
+    @Test
+    fun `it should delete a subscription`() {
+        val subscriptionId = "urn:ngsi-ld:Subscription:04".toUri()
+
+        every { neo4jRepository.getRelationshipTargetOfSubject(any(), any()) } returns null
+        every { entityService.deleteEntity(any()) } returns Pair(2, 1)
+
+        subscriptionHandlerService.deleteSubscriptionEntity(subscriptionId)
+
+        verify {
+            neo4jRepository.getRelationshipTargetOfSubject(
+                eq(subscriptionId),
+                JsonLdUtils.EGM_RAISED_NOTIFICATION.toRelationshipTypeName()
+            )
+        }
+        verify { entityService.deleteEntity(eq(subscriptionId)) }
+
+        confirmVerified()
```

Comment:
I'm really not sure that calling `confirmVerified()` without any mock passed as arguments actually does something